### ISSUE TITLE
Transformation of .shp files with Proj library

### ISSUE THREAD

**Issue by A-Nesse1** [qgis/QGIS#56343]
### What is the bug or the crash?

Transformation of shape files between EPSG 27391 to EPSG 25832 is not correct. If I use Json or DWG format it works fine.  I am using the Proj library for transformation.

### Steps to reproduce the issue

[EPSG-27391.zip](https://github.com/qgis/QGIS/files/14277365/EPSG-27391.zip)
[correct 27391 and 25832-GeoJson.zip](https://github.com/qgis/QGIS/files/14277381/correct.27391.and.25832-GeoJson.zip)

### Versions

3.34.3

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [X] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

**Comment by m-kuhn** [comment]
Can you add the precise steps to reproduce?

**Comment by A-Nesse1** [comment]
Yes I can.  I have attached a shape file and a GeoJson file in EPSG27391, then you can try to transform it in Q-GIS to EPSG 26832 or to EPSG 32632.  You will then see the differences.


Anders


Anders Nesse
Phone: +47 45 40 15 79  |  ***@***.***

Norconsult Digital AS
Kjørboveien 22, NO-1337 Sandvika
Tel: +47 67 57 10 00  |  www.norconsult.no

Every day we improve everyday life

**Comment by m-kuhn** [comment]
Can you add a screencast (or a few screenshots) that show _how_ you transform? You particularly mention the proj library, so I wonder how you explicitly invoke or avoid that.

**Comment by A-Nesse1** [comment]
Attached is my workflow in word format and EPSG27391 files in Shape and GeoJson.

If you try them, you will hopefully see that the transformation are different for .shp and .json

Anders

Anders Nesse
Phone: +47 45 40 15 79  |  ***@***.***

Norconsult Digital AS
Kjørboveien 22, NO-1337 Sandvika
Tel: +47 67 57 10 00  |  www.norconsult.no

Every day we improve everyday life

**Comment by m-kuhn** [comment]
@A-Nesse1 It looks like your attachment didn't make it through, ideally you upload it directly on the github issue web interface.
